.github/workflows/build.yml

```diff
@@ -471,6 +471,9 @@ jobs:
       - name: Package (Windows, installer)
         if: runner.os == 'Windows'
         run: |
+          New-Item -Name NSISPlugins -ItemType Directory
+          Invoke-Webrequest https://github.com/negrutiu/nsis-nscurl/releases/latest/download/NScurl.zip -OutFile NSISPlugins\NScurl.zip
```

Comment:
At minimum we need to pin this to a specific release, as it's currently introducing a clear supply chain issue. Checksum verification would be good as well if you're up to it

Reply:
Fair.